sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1142,20 +1138,14 @@ public partial class JsonObjectSerializer : Azure.Core.Serialization.ObjectSeria
         public override System.Threading.Tasks.ValueTask SerializeAsync(System.IO.Stream stream, object? value, System.Type inputType, System.Threading.CancellationToken cancellationToken) { throw null; }
         public override System.Threading.Tasks.ValueTask<System.BinaryData> SerializeAsync(object? value, System.Type? inputType = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
     }
-    public partial class ModelJsonConverter : System.Text.Json.Serialization.JsonConverter<Azure.Core.Serialization.IJsonSerializable>
+    public partial class ModelJsonConverter : System.Text.Json.Serialization.JsonConverter<Azure.Core.Serialization.IModelSerializable>
```

Comment:
Re: naming:
[.NET precedent in naming JsonConverters](https://github.com/dotnet/runtime/tree/main/src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Converters/Value) would drop the `Json` from the name, but I wonder if that would be too terse for a public type?  Would `AzureModelConverter` be reasonable?

Also, curious whether this needs to be public - are there scenarios where customers would want to pass this converter to their own JsonSerializer.Serialize/Derialize methods?

Reply:
They must pass it in otherwise they will not get our internal serialization logic.  This is how they opt into our conversion logic as well as access the additional properties like `Version` or `IgnoreAdditionalProperties`